lib/logstash/codecs/cef.rb

```diff
@@ -303,29 +220,33 @@ def handle(data, &block)
     message = unprocessed_data
 
     # Try and parse out the syslog header if there is one
-    if (cef_version = event.get('cefVersion')).include?(' ')
+    if (cef_version = event.get(@header_fields[0])).include?(' ')
```

Comment:
To improve readability I would suggest to extract `@header_fields[0]` in local variable perhaps named `cef_version_field_name`